aspnetcore/signalr/javascript-client/sample/chat.js

```diff
@@ -0,0 +1,20 @@
+﻿const transportType = signalR.TransportType.WebSockets;
+const logger = new signalR.ConsoleLogger(signalR.LogLevel.Information);
+const connection = new signalR.HubConnection('/chathub', 
+                   { transport: transportType, logger: logger });
```

Comment:
As I said above, drop the `transport` option. We don't recommend it unless you are a more advanced user.